Test failure in main branch possibly with jinja2 import

We have detected the following test failure:

https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-python%2Fapm-agent-python-mbp/detail/main/27/pipeline/185

Seems as the return code may be coming from this line:

`[2022-03-24T15:09:10.160Z] make: *** [Makefile:14: test] Error 5`

There is a warning about a jinja import, but it's not clear whether or not it's the true cause of this issue:

```
[](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-python%2Fapm-agent-python-mbp/detail/main/27/pipeline/185#step-1454-log-174)[](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-python%2Fapm-agent-python-mbp/detail/main/27/pipeline/185#step-1454-log-175)[](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-python%2Fapm-agent-python-mbp/detail/main/27/pipeline/185#step-1454-log-176)[2022-03-24T15:09:09.899Z] =============================== warnings summary ===============================

[2022-03-24T15:09:09.899Z] tests/instrumentation/jinja2_tests/jinja2_tests.py:42
[2022-03-24T15:09:09.899Z]   /app/tests/instrumentation/jinja2_tests/jinja2_tests.py:42: PytestUnknownMarkWarning: Unknown pytest.mark.jinja2 - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html[](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-python%2Fapm-agent-python-mbp/detail/main/27/pipeline/185#step-1454-log-177)
[2022-03-24T15:09:09.899Z]     pytestmark = [pytest.mark.jinja2]
```

Could somebody please investigate this failure? Thanks.

--- a/setup.cfg
+++ b/setup.cfg
@@ -161,6 +161,7 @@ markers =
     httpx
     prometheus_client
     sanic
+    jinja2
 
 [isort]
 line_length=120

--- a/tests/requirements/reqs-flask-1.0.txt
+++ b/tests/requirements/reqs-flask-1.0.txt
@@ -1,3 +1,4 @@
+jinja2<3.1.0
 Flask>=1.0,<1.1
 MarkupSafe<2.1
 itsdangerous==2.0.1

--- a/tests/requirements/reqs-flask-1.1.txt
+++ b/tests/requirements/reqs-flask-1.1.txt
@@ -1,3 +1,4 @@
+jinja2<3.1.0
 Flask>=1.1,<1.2
 MarkupSafe<2.1
 itsdangerous==2.0.1

